dsbridge/src/main/java/wendu/dsbridge/DWebView.java

```diff
@@ -216,7 +215,15 @@ private void complete(String retValue, boolean complete) {
                         Log.e("SynWebView", error);
                     }
                 } catch (Exception e) {
-                    evaluateJavascript(String.format("alert('ERROR! \\nCall failed：Function does not exist or parameter is invalid［%s］')", e.getMessage()));
+                    JSONObject errObject = new JSONObject();
+                    try {
+                        errObject.put("message", e.getMessage());
+                        errObject.put("method", methodName);
+                        errObject.put("args", args);
```

Comment:
let ensure that methondName and args will always not null

Reply:
if they are `null`, could `errObject` be successfully serialized?

Reply:
i tested it, if the value is `null`, that key won't be included in the final json string:

```java
Log.d("test json null", new JSONObject().put("haha", null).put("name", "chun").toString()); 

=> {"name": "chun"}
```